src/client/app/translations/data.ts

```diff
@@ -1368,114 +1371,115 @@ const LocaleTranslationData = {
 		"per.second": "Por segundo",
 		"projected.to.be.used": "proyectado para ser utilizado",
 		"radar": "Radar",
-		"radar.lines.incompatible": "Estos medidores/grupos no están compatibles para gráficos de radar",
-		"radar.no.data": "No hay lecturas:<br>probablemente el rango de los datos esté fuera del rango de lecturas de los medidores/grupos",
+		"radar.lines.incompatible": "Estos medidores/grupos no son compatibles para gráficos de radares",
+		"radar.no.data": "No hay lecturas:<br>es probable que el rango de los datos esté fuera del rango de lecturas de los medidores/grupos",
 		"rate": "Tasa",
-		"reading": "Lecturas:",
-		"redo.cik.and.refresh.db.views": "Procesando cambios. Esto será un momento.",
-		"readings.per.day": "Lecturas por Día",
+		"reading": "Lectura:",
+		"redo.cik.and.refresh.db.views": "Procesando los cambios. Esto tardará un momento.",
+		"readings.per.day": "Lecturas por día",
 		"redraw": "Redibujar",
 		"remove": "Eliminar",
 		"restore": "Restaurar",
-		"return.dashboard": "Return To Dashboard\u{26A1}",
+		"return.dashboard": "Regresar al panel principal",
 		"role": "Rol",
-		"save.all": "Guardar todos",
-		"save.map.edits": "Guardar las ediciones del mapa",
-		"save.meter.edits": "Guardar ediciones del medidor",
-		"save.role.changes": "Guardar cambios de rol",
+		"save.all": "Guardar todo",
+		"save.map.edits": "Guardar las ediciones al mapa",
+		"save.meter.edits": "Guardar las ediciones al medidor",
+		"save.role.changes": "Guardar los cambios de rol",
 		"second": "Segundo",
 		"select.groups": "Seleccionar grupos",
 		"select.map": "Seleccionar mapa",
-		"select.meter.type": "Seleccionar Tipo de Medidor",
+		"select.meter.type": "Seleccionar tipo de medidor",
 		"select.meter": "Seleccionar medidor",
 		"select.meter.group": "Seleccionar medidor o grupo para hacer gráfico",
 		"select.meters": "Seleccionar medidores",
 		"select.unit": "Seleccionar unidad",
 		"show": "Mostrar",
 		"show.grid": "Mostrar rejilla",
-		"show.options": "Opciones de desacoplamiento",
+		"show.options": "Mostrar opciones",
 		"sort": "Ordenar",
 		"submit": "Enviar",
 		"submitting": "Enviando",
-		"submit.changes": "Enviar cambios",
-		"submit.new.user": "Entregar un nuevo usario",
+		"submit.changes": "Ingresar los cambios",
+		"submit.new.user": "Ingresar un nuevo usario",
 		"the.unit.of.meter": "La unidad del medidor",
 		"this.four.weeks": "Estas cuatro semanas",
-		"timezone.no": "no zona horaria",
+		"TimeSortTypes.decreasing": "decreciente",
+		"TimeSortTypes.increasing": "creciente",
+		"TimeSortTypes.meter": "valor del medidor o predeterminado",
+		"true": "Verdad",
```

Comment:
By the way OED does the ordering, this should not have moved in the file. It turns out the English one is in the wrong place. Could you put this back and fix the English placement?